packages/user_packages/README.md

```diff
@@ -0,0 +1,23 @@
+# Liota User Packages
+
+Basic user packages which we believe should exist on the edge system to publish the basic health stats to Pulse IoT Control Center
+
+* general_edge_system.py
+
+The sample package contains specifications of GeneralEdgeSystem, replace the "EdgeSystemName" with the logic to auto-generate the unique name everytime this package
+is loaded based on the user flow. Unique identifier can be from the system(MAC address) or devices to generate the unique name.
+
+* iotcc_mqtt.py
+
+This is a sample user package which creates a IoTControlCenter DCC object and registers edge system on
+IoTCC over MQTT Protocol to acquire "registered edge system", i.e. iotcc_edge_system. This package has dependency on credentials package
+which is pre-loaded during the installation in order to provide the required credentials and configuration parameters in the liota package manager registry.
+System Properties which are pre-loaded in the registry during installation are required to be set for the registered edge system before the object is stored in the package manager registry,
+all the devices will internally inherit the the system properties from the registered edge system.
+The properties for the edge system can be set as 'key:value' pair, you can also set the location by passing the
+'latitude:value' and 'longitude:value' as properties in the user package.
+
+* iotcc_mqtt_edge_system_stats.py
+
+This is a sample user package to publish the basic edge system stats which we believe are required to
+monitor the health status of the edge system from Pulse IoT Control Center.
```

Comment:
Shouldn't we keep iotcc_mqtt_ram.py?
It should be a good example to show users can write a device liota package, since this is the most important parts for OTs.
We can also point out that set_system_properties() is not needed for metrics, although inside iotcc_mqtt_edge_system_stats.py we also have create metrics procedures.
Just in somewhere, we can point it out to users.

Reply:
It doesn't make sense to have iotcc_mqtt_ram.py as RAM is not an actual device so as per Greg's suggestion I have removed it as there are plenty other packages which can be pointed out to explain device creation in Liota.

Reply:
for liota_lite, which one could be a device creation example? Or they can refer codes in packages/examples?

Reply:
Agree. The RAM device was always kinda weird. Good to remove it.